libshaderc_util/src/spirv_tools_wrapper.cc

```diff
@@ -72,15 +45,21 @@ bool SpirvToolsAssemble(const string_piece assembly, spv_binary* binary,
   *binary = nullptr;
   errors->clear();
 
-  const bool result =
+  const bool success =
```

Comment:
Ok. Good.   I get the feeling that someday I'll add a way to do binary parsing based where the input binary is presented as begin/end forward iterators.  :-)